src/service/exercise/EvaluationService.ts

```diff
@@ -45,4 +45,14 @@ export class EvaluationService {
     const evaluationUpdated = await evaluationRepository.update(id, evaluation)
     return evaluationUpdated
   }
+
+  public async deleteEvaluation (id) {
+    const evaluationRepository = getRepository(Evaluation)
+    const evaluationDeleted = await evaluationRepository.delete(id)
+
+    if (evaluationDeleted.affected === 0) {
+      throw new HttpError('Evaluation not found with: ' + id, HttpStatusCode.BAD_REQUEST)
```

Comment:
por ser um not found, será que o `HttpStatusCode` não seria 404 ao inves de 400?

Reply:
verdade nic, alterado! obrigada